kythe/go/platform/kcd/kcd.go

```diff
@@ -244,6 +246,10 @@ type Unit interface {
 	// Digest produces a unique string representation of a unit sufficient to
 	// serve as a content-addressable digest.
 	Digest() string
+
+	// ResolveVName looks up and returns the VName for the given file path
```

Comment:
Resolve means something kinda different internally.  Can we name this LookupVName?

Reply:
Done.